content/02.introduction.md

```diff
@@ -67,14 +72,30 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 *Presentation of COVID-19:* 
 Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors...
 
-In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
+In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10/ggnxb3] described the clinical features of patients infected with SARS-CoV-2 including lower respiratory tract infection with fever, dry cough, and dyspnea. 
+[@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common which suggests that the virus targets cells located in the lower respiratory tract. 
+Recent reports suggest that symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation. 
+[@doi:10/ggnxb3] also found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
 Mortality also appeared to be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
 They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
```

Comment:
Replace "They" with whoever "They" is here. I assume you mean the group who published the study but the reference is three sentences away at this point so it is not easy to understand who "they" is referring to.

Reply:
What is the significance of this finding? Needs to be put into context.

Reply:
These sentences were already there when I added to the review (I didn't write it), but I think this last sentence was meant to imply that patients might be contagious for up to 37 days. I agree, it doesn't have much context. I'm new to GitHub - is there some way to tell who initially wrote it?

Reply:
Yes, we can see who wrote it by browsing to the introduction file at https://github.com/greenelab/covid19-review/blob/master/content/02.introduction.md and then clicking the "Blame" button at the top.  Despite the negative sounding connotation, it is a helpful way to see who originally wrote or edited something to get clarification from them.  In this case the text is from #164 so @dziakj1 could help clarify the phrasing here.